docs/get-clickhouse-docs.sh

```diff
@@ -27,5 +29,27 @@ else
     exit 1
     ;;
   esac
+
+  if [ -n "$2" ]; then
+    set_git_hook="$2"
+  elif [ -z "$1" ]; then
+    read -rp "Would you like to setup git hook for automatic update? (y|n): " set_git_hook
+  fi
+
+  if [ "$set_git_hook" = "y" ]; then
+    hook_command="$(pwd)/pull-clickhouse-docs-hook.sh $UPDATE_PERIOD_HOURS"
+    hook_file=$(realpath "$(pwd)/../.git/hooks/post-checkout")
+    already_have=$(grep -Faq "pull-clickhouse-docs-hook.sh" "$hook_file" 2>/dev/null && echo 1 || echo 0)
```

Comment:
It's strange style-check is not complaining, now the `already_have` var is not user in the script